tests/queries/0_stateless/02322_sql_insert_format.sql

```diff
@@ -0,0 +1,10 @@
+-- Tags: no-parallel
+
+select number as x, number % 3 as y, 'Hello' as z from numbers(5) format SQLInsert;
```

Comment:
What will be in case when we don't specify aliases? Will the output be like this? 
```
INSERT INTO table (number, modulo(number, 3), 'Hello') VALUES (0, 0, 'Hello'), (1, 1, 'Hello'), (2, 2, 'Hello'), (3, 0, 'Hello'), (4, 1, 'Hello');
```

Reply:
I don't know how to force user to use aliases without AST hacks...